[2025-03-07T12:59:44.303+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2025-03-07T12:59:41.637486+00:00 [queued]>
[2025-03-07T12:59:44.310+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2025-03-07T12:59:41.637486+00:00 [queued]>
[2025-03-07T12:59:44.310+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T12:59:44.321+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2025-03-07 12:59:41.637486+00:00
[2025-03-07T12:59:44.325+0000] {standard_task_runner.py:57} INFO - Started process 4536 to run task
[2025-03-07T12:59:44.327+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'manual__2025-03-07T12:59:41.637486+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmp6k23chiw']
[2025-03-07T12:59:44.330+0000] {standard_task_runner.py:85} INFO - Job 39: Subtask insert_into_table
[2025-03-07T12:59:44.372+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2025-03-07T12:59:41.637486+00:00 [running]> on host afc6f8faa09c
[2025-03-07T12:59:44.441+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-07T12:59:41.637486+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-07T12:59:41.637486+00:00'
[2025-03-07T12:59:44.441+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2025-03-07', 'dag_with_postgres_operator_v03')
        
[2025-03-07T12:59:44.447+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T12:59:44.457+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T12:59:44.460+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2025-03-07', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-03-07T12:59:44.463+0000] {sql.py:427} INFO - Rows affected: 1
[2025-03-07T12:59:44.472+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20250307T125941, start_date=20250307T125944, end_date=20250307T125944
[2025-03-07T12:59:44.499+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T12:59:44.514+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
